src/software/gui/standalone_simulator/widgets/standalone_simulator_draw_function_visualizer.cpp

```diff
@@ -21,6 +21,22 @@ void StandaloneSimulatorDrawFunctionVisualizer::setStandaloneSimulator(
     standalone_simulator = simulator;
 }
 
+void StandaloneSimulatorDrawFunctionVisualizer::keyPressEvent(QKeyEvent* event)
+{
+    if (event->key() == Qt::Key_R)
+    {
+        r_clicked = true;
+    }
+}
+
+void StandaloneSimulatorDrawFunctionVisualizer::keyReleaseEvent(QKeyEvent* event)
+{
+    if (event->key() == Qt::Key_R)
+    {
+        r_clicked = false;
```

Comment:
I don't think we need to stop rotating once the R key is released. It's find to stop the rotation as soon as the mouse is released

Reply:
Both shift click and ctrl click work this way, so I'll change it to be consistent